AesopTest/List.lean

```diff
@@ -186,6 +186,7 @@ theorem subset_trans {l₁ l₂ l₃ : List α} : l₁ ⊆ l₂ → l₂ ⊆ l
 -- END PRELUDE
 
 instance unique_of_is_empty [IsEmpty α] : Unique (List α) := by
+  constructor
```

Comment:
This test now fails without adding `constructor`.

@JLimperg, would it be okay if I left this to you to look into?